src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,175 @@
+            System.out.println("[ERROR] 시간 형식이 일치하지 않습니다.");
+        } catch (DateTimeException e) {
+            System.out.println("[ERROR] 날짜 형식이 일치하지 않습니다.");
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void displayAttendanceEditResult(Attendance originAttendance, LocalDate editDate,
+                                             Attendance updatedAttendance) {
+        if (originAttendance == null) {
+            outputView.displayAttendanceEditWithAbsence(editDate, updatedAttendance.getTime(),
+                    updatedAttendance.determineStatus().getDescription());
+            return;
+        }
+        outputView.displayAttendanceEdit(editDate, originAttendance.getTime(),
+                originAttendance.determineStatus().getDescription(), updatedAttendance.getTime(),
+                updatedAttendance.determineStatus().getDescription());
+    }
+
+    private void checkRecords(LocalDate nowDate) {
+        String name = inputView.readName();
+
+        outputView.displayRecordMessage(name);
+        displayRecords(nowDate, name);
+        outputView.displayPenaltyCount(attendanceBook.calculateAttendanceCount(name, nowDate),
+                attendanceBook.calculateLatenessCount(name, nowDate),
+                attendanceBook.calculateAbsenceCount(name, nowDate)
+        );
+        displayPenaltyStatus(nowDate, name);
+    }
+
+    private void displayPenaltyStatus(LocalDate nowDate, String name) {
+        Penalty penalty = attendanceBook.determinePenaltyStatus(name, nowDate);
+        if (penalty != Penalty.PASS) {
+            outputView.displayPenaltyStatus(penalty.getDescription());
+        }
+    }
+
+    private void displayRecords(LocalDate nowDate, String name) {
+        LocalDate startDate = LocalDate.of(ATTENDANCE_YEAR, ATTENDANCE_MONTH, 1);
+
+        for (LocalDate date = startDate; date.isBefore(nowDate); date = date.plusDays(1)) {
+            displayAttendanceRecord(name, date);
+        }
+    }
+
+    private void displayAttendanceRecord(String name, LocalDate date) {
+        if (Holiday.isHoliday(date)) {
+            return;
+        }
+
+        Attendance attendance = attendanceBook.findAttendance(name, date);
+        if (attendance == null) {
+            outputView.displayAbsenceRecord(date);
+            return;
+        }
+        outputView.displayRecord(date, attendance.getTime(), attendance.determineStatus().getDescription());
+    }
+
+    private void checkExpulsionRiskCrew(LocalDate nowDate) {
+        List<Crew> riskCrewResult = attendanceBook.checkExpulsionRiskCrew(nowDate);
+
+        outputView.displayExpulsionRiskCrewMessage();
+
+        for (Crew crew : riskCrewResult) {
+            outputView.displayExpulsionRiskCrew(crew.getName(), crew.calculateAbsenceCount(nowDate),
+                    crew.calculateLatenessCount(nowDate), crew.determinePenaltyStatus(nowDate).getDescription());
+        }
+    }
```

Comment:
```suggestion
    private void checkExpulsionRiskCrew(LocalDate nowDate) {
        List<Crew> riskCrewResult = attendanceBook.checkExpulsionRiskCrew(nowDate);
        outputView.displayExpulsionRiskCrewList(riskCrewResult, nowDate);
    }
```
출력에 대한 로직들이 컨트롤러에서 for문을 통해 진행되고있네요. 
`OutputView`에 위임해볼까요?
그리고 `check` 네이밍이 수행되는 로직과 맞지 않는 것 같아요.

Reply:
이 부분에서 `outputView`에 `riskCrewResult`를 넘겨주면 `Crew` 객체에 직접 접근할 수 있다는 점이 고민이었습니다. 그래서 컨트롤러에서 뷰와 관련된 처리를 직접 하는 대신, `outputView`를 여러 번 호출하는 방식을 선택했습니다. 

하지만 컨트롤러는 데이터 흐름을 관리하고 비즈니스 로직을 처리하는 역할을 해야 하며, 출력 관련 로직은 `outputView`에서 담당하는 것이 더 적절하다고 판단했습니다! 
또한, `outputView`에서 `Crew` 객체를 직접 다루면 컨트롤러가 단순해지고, 출력 형식이 변경될 때도 수정이 용이하다는 장점이 있습니다.
반영하여 리팩토링하였습니다!


메서드명은 확인 기능에 중점을 두어 `check`라고 네이밍했는데, 검증만 하는 느낌이 드는 것 같습니다! `findAndDisplayExpulsionRiskCrews`로 변경해보았습니다.